src/compiler/crystal/tools/init.cr

```diff
@@ -145,6 +137,20 @@ module Crystal
       args.shift
     end
 
+    def self.validate_name(name)
+      case
```

Comment:
How does this method work without `case name`?

Reply:
What do you mean?

Reply:
I didn't realize you could omit the case's value.  I.e. you have `case` versus `case name`.